src/coreclr/jit/emitarm64.cpp

```diff
@@ -12336,6 +12421,43 @@ void emitter::emitIns_Call(EmitCallType          callType,
     return 0;
 }
 
+/*****************************************************************************
+ *
+ *  Returns the encoding to select the 1/2/4/8 byte elemsize for an Arm64 Sve vector instruction
+ *  This specifically encodes the field 'tszh:tszl' at bit locations '22:20-19'.
+ */
+
+/*static*/ emitter::code_t emitter::insEncodeSveElemsize_tszh_22_tszl_20_to_19(insOpts opt)
+{
+    switch (opt)
+    {
+        case INS_OPTS_SCALABLE_B:
```

Comment:
Problem with these switch statements is they keep getting longer and more SCALABLE types are added. And it's hard to track keeping all the switches up to date. I've just had that exact issue in the PR I'm writing.

Suggestion:

```
emitter::code_t emitter::insEncodeSveElemsize_tszh_22_tszl_20_to_19(emitAttr esize)
{
  switch(esize)
  {
    case EA_1BYTE:
      return 0x080000; // set the bit at location 19
  
    case EA_2BYTE: 
....etc
```

And then it's called as:
```
code |= insEncodeSveElemsize_tszh_22_tszl_20_to_19(optGetElemsize(id->idInsOpt())); // xx
```